### ISSUE TITLE
Implement MathJax in Doubtfire Comments

### ISSUE THREAD

**Issue by jakerenzella** [doubtfire-lms/doubtfire-web#186]
Implement MathJax for comment authorship and comment viewer: https://www.mathjax.org/.

**Comment by TinyServal** [comment]
[KaTeX](https://katex.org/) is probably a much better option at this point since it's a _lot_ faster. Here's a benchmark page that compares both MathJax and KaTeX: https://www.intmath.com/cg5/katex-mathjax-comparison.php

**Comment by ugonj** [comment]
KaTeX's speed is unlikely to be a factor in doubtfire's use case, because it's very unlikely that we will ever need to render lots of equations. The advantage of MathJax is that it has much greater coverage, but again this is unlikely to matter in the use cases within doubtfire. So in the end the decision should go to whatever's easiest to maintain.
